ee/clickhouse/views/groups.py

```diff
@@ -18,11 +18,49 @@ class Meta:
         fields = ["group_type", "group_type_index"]
 
 
-class ClickhouseGroupsView(StructuredViewSetMixin, ListModelMixin, viewsets.GenericViewSet):
+class ClickhouseGroupsTypesView(StructuredViewSetMixin, ListModelMixin, viewsets.GenericViewSet):
```

Comment:
is it okay to leave this types view in this file since it's so small?

Reply:
Yup, they are closely related